src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -86,7 +86,9 @@ public class ParquetRowBuffer extends AbstractRowBuffer<ParquetChunkData> {
   public void setupSchema(List<ColumnMetadata> columns) {
     fieldIndex.clear();
     metadata.clear();
-    metadata.put("sfVer", "1,1");
+    if (!clientBufferParameters.getIsIcebergMode()) {
+      metadata.put("sfVer", "1,1");
```

Comment:
1. from line 101 it looks like we are adding the logical/physical type info to metadata for iceberg mode too (I thought from our slack conv that we weren't adding it in) ?
2. Can we scope this change to only kick in when serialization policy = optimized?